packages/mobile/src/transactions/transferFeedUtils.ts

```diff
@@ -106,7 +108,8 @@ export function getTransferFeedParams(
     recipientInfo
   )
   Object.assign(recipient, { address })
-  const nameOrNumber = recipient?.name || e164PhoneNumber
+  const nameOrNumber =
+    recipient?.name || (recipientHasNumber(recipient) ? recipient.e164PhoneNumber : e164PhoneNumber)
```

Comment:
Nit: don't think you need the ternary. Why not `recipient?.name || recipient?.e164PhoneNumber || e164PhoneNumber`?

Reply:
because of how I declared the `Recipient` type, you need to explicitly check if `e164PhoneNumber` exists before you can call it.

Reply:
How is that different from what `recipient?.e164PhoneNumber` does?